### DISCUSSION TITLE
Limit the number of job executions for cron string repeat option

### DISCUSSION THREAD

**Discussion by sree526** [taskforcesh/bullmq#2646] [Q&A]
**Is your feature request related to a problem? Please describe.**
I want to give cron string in repeat options and limit the number of executions. It's not specified in the documentation as well. Is this not supported now?

**Describe the solution you'd like**
If not supported can we add support for that?

**Describe alternatives you've considered**
Right now I'm maintaining a count of number of times this cron job executed in the db along with the limit and once the count reaches the limit I'm removing this job

**Comment by manast** [comment]
For some reason, it does not appear in the current documentation but you have an option "endDate" where you can specify when the repetitions should end: https://api.docs.bullmq.io/interfaces/v5.RepeatOptions.html

**Reply by sree526** [reply]
But that doesn't help my case. I want to limit the job to 10 executions for example. That limit option is working fine if we are providing every instead of cron
